freqtrade/commands/cli_options.py

```diff
@@ -251,7 +251,8 @@ def __init__(self, *args, **kwargs):
     "spaces": Arg(
         '--spaces',
         help='Specify which parameters to hyperopt. Space-separated list.',
-        choices=['all', 'buy', 'sell', 'roi', 'stoploss', 'trailing', 'protection', 'default'],
+        choices=['all', 'buy', 'sell', 'roi', 'stoploss',
+                 'trailing', 'protection', 'default', 'trades'],
```

Comment:
i'm torn on naming here (default should go last, but that's a small nitpick).

"trades" as space name seems fine at first glance - but the more i think about it makes me feel it's not the right word.

all spaces are optimizing trades to some degree (when to enter, when to exit, when to not enter (protections) ...) - so it can be misunderstood quite easily (and it had me confused while looking through the rest of the changes, too).

We're optimizing trade count (or trade slots, or max_open_trades) - so i find trades almost too generic. 

(before acting/changing - let me know your thoughts on this!)

Reply:
Yeah it felt to disruptive to use a multi-word arg name, since they were all one word lol. 
While `trade_slots` or `trade_count` are good names (and better alternatives to `trades`, since they are more explicative and specific), they may be too distant from the parameter that we are optimizing: `max_open_trades`.
So `max_open_trades` or `open_trades` will be the best options in my opinion.
I would also consider `mot`, which also doesn't violate the one word rule :D

Reply:
Sorry @xmatthias , I didn't explain it well in the last message. I was talking about the `trades` space name, regarding this review. I haven't change it yet, so one would call `--space trades` to hyperopt `max_open_trades`.

Reply:
i'm aware of this - and lacking any idea on a better term - i don't think we should change it again at this point.